Allow empty string for filtering (#1120)
sonata-project/SonataDoctrineORMAdminBundle: docs/reference/filter_field_definition.rst

        }
    }

StringFilter
------------

The string filter has additional options:

> * ``case_sensitive`` - set to ``false`` to make the search case insensitive;

Reviewer: All of these options should indicate its default value or its default behavior.

> * ``trim`` - use one of ``Sonata\DoctrineORMAdminBundle\Filter\TRIM_*`` to control the clearing of spaces in the value. As default used ``Sonata\DoctrineORMAdminBundle\Filter\TRIM_BOTH``;

Reviewer: ```suggestion
* ``trim`` - use one of ``Sonata\DoctrineORMAdminBundle\Filter\TRIM_*`` constants to control the clearing of blank spaces around the value. By default ``Sonata\DoctrineORMAdminBundle\Filter\TRIM_BOTH`` is used;
```
